app/scripts/directives/layout.dir.js

```diff
@@ -6,7 +6,7 @@ angular.module('yourStlCourts').directive('layout', function () {
 
     link: function ($scope, element) {
       $scope.$on('$stateChangeStart', function(event, toState){
-        if (toState.name === 'home' || toState.name === 'ticketSearch' || toState.name === 'about' || toState.name === 'help' || toState.name === 'info') {
+        if (toState.name === 'home' || toState.name === 'ticketSearch' || toState.name === 'about' || toState.name === 'help' || toState.name === 'info' || toState.name === 'error') {
```

Comment:
This line is getting a little ridiculous. I think we might want to move to something like a 'const statesToUseBackground = [...] and if (statesToUseBackground.includes(toState.name)) or something. Just to clean it up.